store/remote_test.go

```diff
@@ -46,19 +48,26 @@ func TestNewRemote(t *testing.T) {
 	if err != nil {
 		t.Fatalf("unexpected error reading index: %v", err)
 	}
+
 	if !ok {
 		t.Fatalf("unexpected index not found")
 	}
+
 	if nb.BlobKey != data {
 		t.Fatalf("bad data returned from store: got %v, want %v", nb.BlobKey, data)
 	}
 
 	// Get a remote with a different URI
 	nc, ok, err := s.GetRemote(u2)
-	// Should get an error, since the URI shouldn't be present in the table
+	if err == nil {
+		// Should get an error, since the URI shouldn't be present in the table
+		t.Fatalf("should have received error")
```

Comment:
I'm saying that `ok != true && err == nil` isn't necessarily a bug (which is what you're suggesting right?)